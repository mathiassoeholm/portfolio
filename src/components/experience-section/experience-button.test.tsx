import React from 'react'
import ExperienceButton from './experience-button'
import { render } from 'react-testing-library'


describe('experience-button', () => {
  it('renders correctly while selected', () => {

    const { container } = render(
      <ExperienceButton text="Button" selected onClick={() => {}} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })

  it('renders correctly while not selected', () => {

    const { container } = render(
      <ExperienceButton text="Button" onClick={() => {}} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
