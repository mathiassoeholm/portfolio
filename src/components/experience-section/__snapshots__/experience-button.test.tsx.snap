// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`experience-button renders correctly while not selected 1`] = `
<button
  class="pf-experience-button"
  style="background-color: rgba(255, 255, 255, 0); padding-left: 0.2em;"
>
  Button
</button>
`;

exports[`experience-button renders correctly while selected 1`] = `
<button
  class="pf-experience-button"
  style="background-color: rgba(255, 255, 255, 0.1); padding-left: 1em;"
>
  Button
</button>
`;
